src/Controller/LevelEditorController.php

```diff
@@ -53,4 +53,14 @@ private function parsePost(array &$level, array &$cells, array &$errors): void
             }
         }
     }
+    public function delete(int $id)
+    {
+        $levelManager = new LevelManager();
+        $level = $levelManager->selectOneById($id);
+        if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+            $levelManager = new LevelManager();
+            $levelManager->delete($level);
```

Comment:
strange to have these delete lines in a method returning an edit template. moreover, your delete method asks for an id as parameter, not a full Level. I guess what you want is more something like this:

    public function edit(int $id)
    {
        $levelManager = new LevelManager();
        $levelManager->selectOneById($id);
        // handle level edition, if it's a "submit" request
        return $this->twig->render('Editor/edit.html.twig');
    }

    public function delete(int $id)
    {
        $levelManager = new LevelManager();
        $levelManager->delete($id);
        // maybe redirect to an other page
    }